### PR TITLE
Added tasks 1327, 1341, 1478, 1517, 1633

### PR DESCRIPTION
**Pull request by ThanhNIT**

### CODE REVIEW

File: src/test/java/g1501_1600/s1529_minimum_suffix_flips/SolutionTest.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+class SolutionTest {
+    @Test
+    void test() {
+        assertThat(new Solution().decode(""), equalTo(""));
```
Comment (javadev) [inline src/test/java/g1501_1600/s1529_minimum_suffix_flips/SolutionTest.java]: Please add tests

```kotlin
internal class SolutionTest {
    @Test
    fun minFlips() {
        assertThat(Solution().minFlips("10111"), equalTo(3))
    }

    @Test
    fun minFlips2() {
        assertThat(Solution().minFlips("101"), equalTo(3))
    }

    @Test
    fun minFlips3() {
        assertThat(Solution().minFlips("00000"), equalTo(0))
    }
}
```

File: src/main/java/g1501_1600/s1529_minimum_suffix_flips/Solution.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package g1501_1600.s1529_minimum_suffix_flips;
+
+// #Medium #String #Greedy
+
+public class Solution {
+    public String decode(String value) {
```
Comment (javadev) [inline src/main/java/g1501_1600/s1529_minimum_suffix_flips/Solution.java]: Please add solution 

```kotlin
class Solution {
    fun minFlips(target: String): Int {
        var flipCount = target[0].code - 48
        var prev = target[0]
        for (ch in target.toCharArray()) {
            if (ch != prev) {
                flipCount++
                prev = ch
            }
        }
        return flipCount
    }
}
```

Comment (ThanhNIT) [reply src/main/java/g1501_1600/s1529_minimum_suffix_flips/Solution.java]: Solutions 1478 and 1529 are already existed

**Comment by javadev** [comment]
✅

**Pull request merged**